db/providers/notesProvider.js

```diff
@@ -0,0 +1,42 @@
+import dotenv from "dotenv";
+
+import Note from "../../models/notes";
+import { LIMIT_NUMBER } from "../../config/constants";
+import { postfix } from "../../utils/environment";
+
+dotenv.config();
+
+export const getNotes = async ({ filter }) => {
+  const notes = await Note.find(filter)
```

Comment:
You should also set as a filter:
deleted: false. Because we don't want to get deleted notes! Here you should update props for `find` method